requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy
 xarray
-dask[array] == 2021.02.0
-distributed == 2021.02.0
+dask[array]
```

Comment:
Looks like this undoes earlier PR?

Reply:
I was thinking it would go in after #482, but actually it is independent. I've rebased to remove #482 from this PR.